src/lsp/main.ts

```diff
@@ -397,6 +398,18 @@ async function openLogFile(): Promise<void> {
     }
 }
 
+export async function getClojuredocs(symName: string, symNs: string): Promise<any> {
+    if (serverVersion > '2021.10.20-16.49.47') {
```

Comment:
How does this `>` operator work when checking against a string?

Reply:
It seems that even if this check worked as it looks it was intended, there's a chance that `null` would be returned from this function just because the version was not a desired/capable version, and then the message [here](https://github.com/BetterThanTomorrow/calva/blob/7d33d621265e9d9f56a0c1ba3f7ac0f307ceaabf/src/clojuredocs.ts#L190) would be misleading, because `clojureDocs` would be `null` but symbol maybe would exist. It would say `Only Clojure core (ish) symbols supported` when really the version was not a desired/capable version.

Reply:
It's ”dictionary” order, character by character. https://javascript.info/comparison#string-comparison This type of check works because clojure-lsp has the kind of version string that it has. For other schemes, like Calva's, we would rather bring in a semver library, I think.

As for the message: good catch! I originally wrote that for when there were only direct commands fetching clojuredocs. Now when it happens on hover, it makes no sense to show a message. And we don't actually, so probably the `message` field should be removed. Which actually puts the whole `NoDocsEntry` construct into question (that might have been a bad idea all the time, I don't know 😄).

Reply:
Ah, thanks for educating me on that comparison!